test/cmpd_dtransform.c

```diff
@@ -0,0 +1,139 @@
+/* * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * *
+ * Copyright by The HDF Group.                                               *
+ * Copyright by the Board of Trustees of the University of Illinois.         *
+ * All rights reserved.                                                      *
+ *                                                                           *
+ * This file is part of HDF5.  The full HDF5 copyright notice, including     *
+ * terms governing use, modification, and redistribution, is contained in    *
+ * the files COPYING and Copyright.html.  COPYING can be found at the root   *
+ * of the source code distribution tree; Copyright.html can be found at the  *
+ * root level of an installed copy of the electronic HDF5 document set and   *
+ * is linked from the top-level documents page.  It can also be found at     *
+ * http://hdfgroup.org/HDF5/doc/Copyright.html.  If you do not have          *
+ * access to either file, you may request a copy from help@hdfgroup.org.     *
+ * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * */
```

Comment:
The HDF Group copyright header has changed, and the line "* Copyright by the Board of Trustees of the University of Illinois." applies only to files created before 2007.
The text in the paragraph has changed and the link that is now for the COPYING file also needs updating in all of the HDF5 source files, which will be happening within a few days, so it isn't necessary to make changes for this pull request.  Once that update is done, this header will be replaced with the following:
/* * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * *  
 * Copyright by The HDF Group.                                                                 *
 * All rights reserved.                                                                                   *
 *                                                                                                                  *
 * This file is part of HDF5.  The full HDF5 copyright notice, including       *
 * terms governing use, modification, and redistribution, is contained in  *
 * the COPYING file, which can be found at the root of the source code   *
 * distribution tree, or in https://www.hdfgroup.org/licenses.                    *
 * If you do not have access to either file, you may request a copy from   *
 * help@hdfgroup.org.                                                                                 *
 * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * */

Reply:
I have updated the source header to the latest version.

Reply:
Thanks!